EPRN.Portal/Services/PRNService.cs

```diff
@@ -221,13 +222,28 @@ public async Task<ViewPRNViewModel> GetViewPrnViewModel(string reference)
             return _mapper.Map<ViewPRNViewModel>(dto);
         }
 
-        public async Task<ActionPrnViewModel> GetActionPrnViewModel(int id)
+        public async Task<DestinationPrnViewModel> GetActionPrnViewModel(int id)
         {
-            //TODO: This will need to be populated by the journey so we can return to it.
-            return new ActionPrnViewModel
-            {
+            var dto = await _httpPrnsService.GetPrnReference(id);
+            var actionPrnViewModel = new DestinationPrnViewModel 
+            { 
                 Id = id
             };
+            return actionPrnViewModel;
+        }
+
+        public async Task SaveDraftPrn(DestinationPrnViewModel destinationPrnViewModel)
+        {
+            if (destinationPrnViewModel == null)
+                throw new ArgumentNullException(nameof(destinationPrnViewModel));
+
+            if (destinationPrnViewModel.DoWithPRN == null)
+                throw new ArgumentNullException(nameof(destinationPrnViewModel.DoWithPRN));
+
+            destinationPrnViewModel.Reason = "Update";
```

Comment:
Yeah, let's get rid of reason being a required field.

Reply:
Done.